Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION=3.12
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+COPY . ./
+
+FROM python:${PYTHON_VERSION}-slim
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=base /app .
+
+RUN pip install --upgrade pip && \
+pip install -r requirements.txt
+
+EXPOSE 8080
+
+ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"]
```

Comment:
Step of migration should be as RUN stage 
![image](https://github.com/user-attachments/assets/ef539ac7-6ab0-4e1b-9c10-730b33ef81ce)

Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/23911762-28a5-425a-8b8c-57f5e049567a)